pylint_odoo/checkers/no_modules.py

```diff
@@ -371,11 +377,21 @@ def colon_list_to_dict(self, colon_list):
         """
         return dict([item.split(":") for item in colon_list])
 
+    def replace_and_first_lower(self, string):
```

Comment:
`string` name variable is not a real name of variable. In fact, is a name of type of variable.
Could you use a real name of variable?